2023/sg-election-votes/main.js

```diff
@@ -0,0 +1,117 @@
+// For API and chart documentation please look here:
+// https://www.highcharts.com/demo
+
+(function (H) {
+    H.wrap(
+        H.Legend.prototype,
+        'colorizeItem',
+        function (proceed, item, visible) {
+            item.legendColor = item.options.legendColor;
+            proceed.apply(this, Array.prototype.slice.call(arguments, 1));
+        }
+    );
+})(Highcharts);
+
+Highcharts.chart('chart-div', {
+    // Setting default colors
+    colors: [
+        '#FFA319',
+        '#800000',
+        '#C16622',
+        '#8F3931',
+        '#8A9045',
+        '#58593F',
+        '#155F83',
+        '#350E20',
+        '#47B5FF',
+        '#FF3399'
+    ],
+
+    // All code for your chart goes here
+    chart: {
+        type: 'column'
+    },
+    title: {
+        text: 'College Council Class of 2027 Election Results',
+        align: 'left'
+    },
+    subtitle: {
+        text: 'The top five candidates were elected.',
+        align: 'left'
+    },
```

Comment:
All of our titles and subtitles are centered. 

You could change the subtitle text to `The top five candidates are elected to office.` I think present-tense makes more sense.